simulation/simple_sensor_simulator/include/simple_sensor_simulator/sensor_simulation/sensor_simulation.hpp

```diff
@@ -114,12 +115,22 @@ class SensorSimulation
     }
   }
 
+  auto attachImuSensor(
+    const double /*current_simulation_time*/,
+    const simulation_api_schema::ImuSensorConfiguration & configuration, rclcpp::Node & node)
+    -> void
+  {
+    imu_sensors_.push_back(std::make_unique<ImuSensor<sensor_msgs::msg::Imu>>(
+      configuration, node.create_publisher<sensor_msgs::msg::Imu>("/sensing/imu/imu_data", 1)));
```

Comment:
Could you update the communication document for this imu topic?
document: https://tier4.github.io/scenario_simulator_v2-docs/developer_guide/Communication/
document source: https://github.com/tier4/scenario_simulator_v2/blob/master/docs/developer_guide/Communication.md